### PR TITLE
:sparkles: Add managed cluster arn for the klusterlet api

### PR DESCRIPTION
**Pull request by suvaanshkumar**
## Summary
This change  is needed to generate md5 hash in klusterlet_controller
## Related issue(s)

Fixes #

### CODE REVIEW

**Review by mikeshng — COMMENTED** [review]
https://github.com/open-cluster-management-io/api/blob/main/docs/development.md

You need to follow some steps to autogen some additional changes.

File: operator/v1/types_klusterlet.go [hunk 1]
```diff
@@ -193,7 +193,8 @@ type AwsIrsa struct {
 	// The arn of the hub cluster (ie: an EKS cluster). This will be required to pass information to hub, which hub will use to create IAM identities for this klusterlet.
 	// Example - arn:eks:us-west-2:12345678910:cluster/hub-cluster1.
 	// +required
-	HubClusterArn string `json:"hubClusterArn"`
+	HubClusterArn     string `json:"hubClusterArn"`
+	ManagedClusterArn string `json:"managedClusterArn"`
```
Comment (mikeshng) [inline operator/v1/types_klusterlet.go:203]: Could you please add a comment to describe what this field is for?

**Review by mikeshng — APPROVED** [review]
/lgtm

File: operator/v1/types_klusterlet.go (hunk 1 above)
Comment (qiujian16) [reply operator/v1/types_klusterlet.go:203]: if this is the required field, you would want a validation for minLength to be 1? Same for HubClusterArn

**Review by mikeshng — APPROVED** [review]
/lgtm

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**